common/lib/xmodule/xmodule/modulestore/tests/test_mongo_call_count.py

```diff
@@ -155,9 +155,9 @@ def _import_course(self, content_store, modulestore):
         # The line below shows the way this traversal *should* be done
         # (if you'll eventually access all the fields and load all the definitions anyway).
         (MIXED_SPLIT_MODULESTORE_BUILDER, None, False, True, 3),
-        (MIXED_SPLIT_MODULESTORE_BUILDER, None, True, True, 38),
-        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, False, True, 38),
-        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, True, True, 38),
+        (MIXED_SPLIT_MODULESTORE_BUILDER, None, True, True, 131),
+        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, False, True, 131),
+        (MIXED_SPLIT_MODULESTORE_BUILDER, 0, True, True, 131),
```

Comment:
Why have the calls increased so drastically from 38 to 131?

Reply:
@ormsbee I think the content scoping is the issue. I've tested locally by setting the scope of settings and the counts remain the same.